### ISSUE TITLE
different sizes of foyer image for stages

### ISSUE THREAD

**Issue by helenvarley** [upstage-org/upstage#897]
i just noticed that on dev-app the two stages shown on the foyer have different sized images. does this mean that the size will depend on the dimensions of the image chosen as cover image? & when there are more than 3 stages on a foyer, will the images be aligned in columns rather than rows? (this is fine - i just want to know what to expect & how to describe it in the manual).
![Screenshot from 2022-04-24 12:29:15](https://user-images.githubusercontent.com/3437324/164972252-e6c851b4-e71a-44c2-a96e-ac0500bc7031.png)

**Comment by nguyenhongphat0** [comment]
Yes stages on foyer will now fit to the dimension of cover image (to prevent these weird blank in the past). What I'm trying to achieve is the masonry grid layout, which would look like this 👇 . What do you think?

![image](https://user-images.githubusercontent.com/29925961/164981942-9dd0d87a-deab-4e6b-91d6-ec05c74f4d9c.png)

At the moment it looks like this (row by row):

<img width="1354" alt="Screen Shot 2022-04-24 at 21 49 17" src="https://user-images.githubusercontent.com/29925961/164982187-b4c3d5ab-0225-438e-8dd5-ae005b150f51.png">

**Comment by helenvarley** [comment]
yes i vote for the masonry grid :sunglasses:

**Comment by helenvarley** [comment]
(adding @elisadcg for design thoughts :smile: )

**Comment by vickis** [comment]
Didn’t we already choose to go down this track - def masonry grid works for
me :) thanks!

**Comment by nguyenhongphat0** [comment]
Here you go:
<img width="1328" alt="Screen Shot 2022-05-05 at 23 58 48" src="https://user-images.githubusercontent.com/29925961/166974715-159be416-afac-46e2-aefa-3c0553620676.png">

**Comment by helenvarley** [comment]
looking great! :+1:

**Comment by vickis** [comment]
love it! :)
